agent_build/package_builders.py

```diff
@@ -405,13 +416,12 @@ def _build_frozen_binary(
 
         # Create package info file. It will be read by agent in order to determine the package type and install root.
         # See '__determine_install_root_and_type' function in scalyr_agent/__scalyr__.py file.
-        package_info_file = self._intermediate_results_path / "package_info.json"
+        install_info_file = self._intermediate_results_path / "install_info.json"
```

Comment:
Here I see the file install_info has .json extension, but in other places is without extension. Or am I missing something?

Reply:
EDIT: Sorry, I needed to refresh to see the latest changes.

Reply:
Sorry, I pushed recently another commit where I addressed this comment https://github.com/scalyr/scalyr-agent-2/pull/829#discussion_r808848373